department-of-veterans-affairs/next-build
# 16767 vet center branch remake

## Description
Relates to #[16767](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16767). (or closes?)
Remake of original PR as branch would not update for some reason

Adds in Components for vet center, hours and facility map
Adds queries, formaters,  and types for vet centers
Handles logic for handling centralized content in formatters

## Testing done
Local visual testing, playwrite tests added, unit tests added

## Screenshots
<img width="1086" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/1fb17c47-08c1-4567-9e8f-b7c01c2688aa">


## QA steps
Verify vet-centers render as expected (ie: http://localhost:3000/pittsburgh-vet-center)
Verify storybook components render

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
